src/network-layer/default/RelayDefaultNetworkLayer.js

```diff
@@ -89,7 +89,8 @@ class RelayDefaultNetworkLayer {
         }
       }).catch(
         error => request.reject(error)
-      )
+      ),
+      request
```

Comment:
Here is : http://jsbin.com/zelerisose/17/edit?js,console
It is what i have in my project right now. The same result.